### PR TITLE
docs: Login Rules Terraform docs

### PR DESCRIPTION
**Pull request by nklaassen**
This PR adds docs for configuring Login Rules via the Terraform provider

Issue: https://github.com/gravitational/teleport/issues/20340

### CODE REVIEW

**Review by alexfornuto — COMMENTED** [review]
Just a few things I noticed. As discussed, I'll wait for a future release to review further.

File: docs/pages/access-controls/login-rules/terraform.mdx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+$ tctl create loginrule-manager.yaml
+role 'loginrule-manager' has been created
+```
+
+(!docs/pages/includes/add-role-to-user.mdx role="loginrule-manager" !)
+
+## Step 2/4. Create a Terraform configuration
+
+Paste the following into a file called `main.tf` to configure the Terraform
+provider and create two example Login Rules:
```
[the remark this answers is not in the document]
Comment (nklaassen) [reply docs/pages/access-controls/login-rules/terraform.mdx]: The only thing they should need to change is their proxy address, reworded to point that out

**Comment by nklaassen** [comment]
Hey @alexfornuto I ran through this guide again on 12.1.5, everything is working for me on this version, this is ready for review

File: docs/pages/access-controls/login-rules/terraform.mdx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+This guide will explain how to:
+
+- Use Teleport's Terraform Provider to deploy Login Rules to your Teleport cluster
+- Edit deployed Login Rules via Terraform
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
```
Comment (alexfornuto) [inline docs/pages/access-controls/login-rules/terraform.mdx:19]: Update for newest version of the tctl partial.

```suggestion
- (!docs/pages/includes/tctl.mdx!)
```

**Review by alexfornuto — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

**Review by espadolini — APPROVED** [review]

**Pull request merged**